torchvision/models/detection/roi_heads.py

```diff
@@ -497,11 +501,11 @@ def postprocess_detections(self, class_logits, box_regression, proposals, image_
 
             # batch everything, by making every class prediction be a separate instance
             boxes = boxes.reshape(-1, 4)
-            scores = scores.flatten()
-            labels = labels.flatten()
+            scores = scores.reshape(-1)
```

Comment:
Yes, but the ONNX/PyTorch's flatten are not equivalents, which is why we have to create a constant in the ONNX subgraph in some cases (here https://github.com/pytorch/pytorch/blob/c864454a8f3bb2e73537a65becc344605765ddcb/torch/onnx/symbolic_opset9.py#L1657) with the desired output shape. So for inputs of different shapes, this can be an issue.
I made a note to update flatten to fix these cases.